miden-crypto/src/utils/mod.rs

```diff
@@ -49,13 +50,26 @@ pub fn bytes_to_hex_string<const N: usize>(data: [u8; N]) -> String {
 
 /// Defines errors which can occur during parsing of hexadecimal strings.
 #[derive(Debug, Error)]
+/// Errors that can occur when decoding hexadecimal strings.
 pub enum HexParseError {
+    /// The input hex string has an incorrect length.
     #[error("expected hex data to have length {expected}, including the 0x prefix, found {actual}")]
-    InvalidLength { expected: usize, actual: usize },
+    InvalidLength {
+        /// The expected length of the input, including the "0x" prefix.
+        expected: usize,
+        /// The actual length of the input.
+        actual: usize,
```

Comment:
nit: same comment as above, i.e., the code is self explanatory so no need for the comments on the fields